R/altdoc_variables.R

```diff
@@ -0,0 +1,38 @@
+.substitute_altdoc_variables <- function(x, filename, path = ".") {
+    x <- gsub("\\$ALTDOC_VERSION",utils::packageVersion("altdoc"), x)
+
+    # DESCRIPTION file
+    fn <- fs::path_join(c(path, "DESCRIPTION"))
+    if (fs::file_exists(fn)) {
```

Comment:
DESCRIPTION always exists in a package, do we want to support other structures?

Reply:
Maybe eventually I can build my personal website with this, but that's not in the cards right now. This is just a safety precaution because `desc` does not fail gracefully.